modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,300 @@
+The process for setting up a basic {productname} 7 instance is the same as {productname} 6.
 
 Most configuration changes in {productname} 7.0 only affect complex use cases, such as custom plugins and customized user interface components.
 
-This chapter describes the settings that require updating prior to migration; and workaround procedures for deprecated, deleted, and updated features.
 
-It set outs the {productname} 7.0 changes that customers using TinyMCE 6 should be aware of as part of upgrading.
+It set outs the {productname} 7.0 changes that customers using {productname} 6 should be aware of as part of upgrading.
 
-NOTE: For support related to migration, please contact https://support.tiny.cloud/hc/en-us/requests/new[Tiny support].  +
+NOTE: For support related to migration, please contact https://support.tiny.cloud/hc/en-us/requests/new[Tiny Support].  +
 Open Source users: please report issues in the https://github.com/tinymce/tinymce/[TinyMCE GitHub Repository].
 
-== Contents
+[[overview]]
+== Overview
 
-* xref:apis[APIs]
-* xref:commands-and-queries[Commands and queries]
+* xref:commands-and-apis[Commands and APIs]
 * xref:options[Options]
 * xref:plugins[Plugins]
-* xref:schema[Schema]
-* xref:ui-and-ux-elements-and-components[UI and UX elements and components]
+* xref:ui-and-ux[UI and UX]
 * xref:security[Security]
 
-== {productname} {productmajorversion} core changes
+[[tinymce-70-core-changes]]
+== {productname} {productmajorversion} core changes.
 
 For additional details on {productname} 7.0 changes, see xref:7.0-release-notes.adoc[{productname} 7.0 release notes].
 
-[[apis]]
-== APIs
 
-...
+[[commands-and-apis]]
+== Commands and APIs 
 
-[[commands-and-queries]]
-== Commands and queries
+This section documents the core changes made to {productname} 7.0.
 
-...
+=== Changed
+
+==== Removed `InsertOrderedList` and `InsertUnorderedList` commands from {productname} core.
+
+Previously, native list commands were executable through various text patterns, leading to undefined browser behavior due to the embedded list code.
+
+[NOTE]
+To utilize these commands, include `plugins: [ 'lists' ]` in the configuration. This action enables the commands previously removed from the core, providing access through the `lists` plugin.
+
+.Example adding list plugin
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+  plugins: [
+    "lists"
+  ],
+  toolbar: "bullist numlist",
+});
+----
+
+[[remove-trailing-brs-configuration-property-was-removed-from-our-domparser-api]]
+==== `remove_trailing_brs` configuration property was removed from our `DomParser` API
+
+The `remove_trailing_brs` configuration property was moved to the serializer.
+
+[TIP]
+While the capability still exists for users that have added this option to their {productname} configuration, it is recommended that customers using our `DomParser` API directly, should migrate to use our `DomSerializer` setting instead.
```

Comment:
```suggestion
[[remove-trailing-brs-configuration-property-was-removed-from-our-domparser-api]]
==== `remove_trailing_brs` option new default and removal from DomParser

 The `remove_trailing_brs` editor option is now defaulted to `+true+`. Also, the `remove_trailing_brs` property was removed from our xref:apis/tinymce.html.domparser.adoc[`DomParser` API]. This means that the `remove_trailing_brs` configuration option does not have an effect when content is inserted into the editor, only when the editor content is retrieved, as the `remove_trailing_brs` property is now only in our xref:apis/tinymce.html.serializer.adoc[`DomSerializer` API].
```
I would add an xref for remove_trailing_brs but it was removed.

Reply:
See [message in clean up PR](https://github.com/tinymce/tinymce-docs/pull/3165/files#r1530168846). I will wait for these to be both resolved before we merge.